### DISCUSSION TITLE
Markdown link button doesn't use selected text as an input

### DISCUSSION THREAD

**Discussion by sudojunior** [community/community#23939] [New to GitHub]
<p>When using the wiki editor, markdown doesn’t pick up any user selected input to put in <em>Link Text</em>. Additionally if link text is entered but no URL is provided, any selected text is deleted and anything typed in the input is lost.</p>
<p>Is this intended behaviour for GitHub Wikis?</p>
<p><img src="https://user-images.githubusercontent.com/8607699/181080325-23156c3e-0374-4ea6-8fac-dae6b1453188.png" alt="image" data-base62-sha1="8xr2Cjn1AgHBgBs7ORVU2hOtP4z" width="609" height="293"></p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/sudojunior">@sudojunior</a> :wave:</p>
<p>This does not appear to represent typical behavior for inserting links in markdown files. This issue has been reported internally. Thanks for bringing this to our attention.</p>

**Reply by hhe** [reply]
It works in the comment WYSIWYG editor and is pretty universally accepted. I should be able to link after typing or type after linking.

**Reply by hhe** [reply]
It also works when opening pull requests, just not in the wiki.
